README.md

```diff
@@ -164,3 +164,9 @@ IDEA must also be manually configured for imports : `Preferences/Settings > Edit
 Then go to `Preferences/Settings > Editor > General > Auto import` and check "Optimize imports on the fly" for Java:
 
 ![image](https://user-images.githubusercontent.com/50145663/135304815-0b51f1da-16d4-4fc7-a278-a13067e78d7c.png)
+
+## (Optional) Built Configuration for Intellij
```

Comment:
I think we can make the section title more precise here. E.g.
```suggestion
## (Optional) Configuring IntelliJ to Build using Maven
```

Reply:
I was considering this too but looking at the general structure of the document, it seemed more appropriate to keep the title of the section neutral:

```markdown
## SonarLint
## Git
## Eclipse Configuration
## Code Style Configuration for Intellij
## (Optional) Build Configuration for Intellij
```

Another option would be to make this a sub-section of `## Intellij Configuration`:

```markdown
## SonarLint
## Git
## Eclipse Configuration
## Intellij Configuration
### Code Style 
### (Optional) Build Configuration
```